scripts/build_and_push_images.sh

```diff
@@ -38,7 +43,7 @@ sed -i 's/deploymentName: webhook/deploymentName: osp-director-operator-controll
 for csv_image in $(cat bundle/manifests/osp-director-operator.clusterserviceversion.yaml | grep "image:" | sed -e "s|.*image:||" | sort -u); do
   base_image=$(echo $csv_image | cut -f 1 -d':')
   tag_image=$(echo $csv_image | cut -f 2 -d':')
-  if ["$base_image:$tag_image" -eq "controller:latest"]; then
+  if [[ "$base_image:$tag_image" == "controller:latest" ]]; then
```

Comment:
double [[ ]] is extended test in bash for pattern matching between two strings, here one [ ]  would be sufficient, anyway no real change to the logic in this case.